docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>"The bash_app requires the type of the task which in this case it is an MPI and the number of cpu_processes (2 MPI ranks where each rank takes 1 core) to run on." -&gt; "The bash_app requires the type of the task and the number of cpu_processes on which to run. In this case, the type of the task is MPI and the number of cpu_processes is 2 MPI ranks, where each rank takes 1 core."</p><p> </p><p>We may want to mention how the use of 2 MPI ranks relates to the worker configuration above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ExaWorks/SDK/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>